src/Scrabble/Board/Board.hs

```diff
@@ -39,7 +39,7 @@ import Scrabble.Board.Square
 
 data Board = Board {
   contents :: (Array Point Square)
-} deriving (Eq, Ord, Generic)
+} deriving (Eq, Ord, Generic, Show)
```

Comment:
Is this necessary for this PR? I think I remember intentionally not deriving Show, and using the showBoard function everywhere that I needed to print a Board. It's much nicer than the default.